templates/books.html

```diff
@@ -0,0 +1,36 @@
+<h2>{{booksShowCtrl.book.title}}</h2>
+
+<div class="row">
+  <div class="col-xs-6">
+    <img ng-src="{{booksShowCtrl.book.image}}" class="img-responsive">
+  </div>
+  <div class="col-xs-6">
+    <p><strong>Author:</strong> {{booksShowCtrl.book.author}}</p>
+    <p><strong>Release Date:</strong> {{booksShowCtrl.book.releaseDate}}</p>
+    <br>
+
+    <!-- buttons to edit/delete -->
+    <a href="#" ng-click="editForm=!editForm" class="btn btn-default">Edit <span class="glyphicon glyphicon-pencil"></span></a>;
+    <a href="#" ng-click="booksShowCtrl.deleteBook(booksShowCtrl.book)" class="btn btn-default">Delete <span class="glyphicon glyphicon-trash"></span></a>
+
+    <!-- edit form -->
+    <div ng-show="editForm">
+      <br>
+      <form ng-show="editForm" ng-submit="booksShowCtrl.updateBook(booksShowCtrl.book)">
+        <div class="form-group">
+          <input type="text" ng-model="booksShowCtrl.book.title" class="form-control" placeholder="Title">
```

Comment:
You should use labels rather than placeholders to describe what input fields are for. See https://www.nngroup.com/articles/form-design-placeholders/